modules/installing-rhv-creating-custom-vm.adoc

```diff
@@ -0,0 +1,179 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_rhv/installing-rhv-default.adoc
+// * installing/installing_rhv/installing-rhv-customizations.adoc
+
+[id="installing-rhv-creating-custom-vm_{context}"]
```

Comment:
@kalexand-rh  This new topic (just added today, 4-17) is a workaround for an unfortunate bug in the 4.4 GA version of the installation program/backend. We expect this to be resolved asap in an upcoming z-stream release. I will remove this topic then.